routes/web.php

```diff
@@ -150,11 +143,6 @@
     // demo helper
     Route::permanentRedirect('demo', '/');
 
-    // blank page, dummy page for external code using Laravel::bootWeb()
```

Comment:
You can't remove this route.  You need to create a controller.

Reply:
It was added by:
https://github.com/librenms/librenms/commit/4e4ceffc0ae5750a647ec7aa946e84e952599b0c#diff-267b660c9508115406d8e6be937c61b3R62

And removed here:
https://github.com/librenms/librenms/commit/c9748b166eb0e16830eb6057e5b962a109eac428#diff-267b660c9508115406d8e6be937c61b3L62

Reply:
Well that explains why every legacy page is being rendered twice!

Reply:
I did some research and it is safe to remove this, sorry about that but I needed to make sure :)